src/Member/MemberUtil.php

```diff
@@ -172,4 +174,49 @@ public function findOrCreate(string $email)
 
         return $member;
     }
+
+    /**
+     * @return \Contao\Model[]|null
+     */
+    public function getActiveGroups(int $memberId, array $options = [])
```

Comment:
This method must always return an array, hence add `: array` and fix the returns of null according.